hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBOptions.java

```diff
@@ -279,11 +351,41 @@ public static synchronized RocksDBOptions instance() {
     public static final ConfigOption<Integer> MAX_MEMTABLES_TO_MAINTAIN =
             new ConfigOption<>(
                     "rocksdb.max_write_buffer_number_to_maintain",
-                    "The total maximum number of write buffers to maintain in memory.",
+                    "The total maximum number of write buffers to maintain in memory " +
+                    "for conflict checking when transactions are used.",
                     rangeInt(0, Integer.MAX_VALUE),
                     0
             );
 
+    public static final ConfigOption<Double> MEMTABLE_PREFIX_BLOOM_SIZE_RATIO =
+            new ConfigOption<>(
+                    "rocksdb.memtable_prefix_bloom_size_ratio",
```

Comment:
rename to `memtable_bloom_size_ratio`: https://github.com/facebook/rocksdb/pull/9453/files#diff-cde52d1fcbcce2bc6aae27838f1d3e7e9e469ccad8aaf8f2695f939e279d7501R369